ambry-router/src/test/java/com.github.ambry.router/GetBlobOperationTest.java

```diff
@@ -1047,6 +1064,7 @@ public void testRangeRequestCompositeBlob() throws Exception {
   public void testEarlyReadableStreamChannelClose() throws Exception {
     for (int numChunksInBlob = 0; numChunksInBlob <= 4; numChunksInBlob++) {
       for (int numChunksToRead = 0; numChunksToRead < numChunksInBlob; numChunksToRead++) {
+        System.out.println("" + numChunksInBlob + " chunks but only read " + numChunksToRead);
```

Comment:
Could you remove this print line? If this is for debugging if the test fails, you can consider adding `numChunksInBlob` and `numChunksToRead` to the assert messages.

Reply:
yes, I will remove them.